tools/extra/pac/fpgabist/dma/fpga_dma_test.c

```diff
@@ -54,6 +61,132 @@ static int err_cnt;
       goto label;\
     } \
   } while (0)
+/*
+ * Global configuration of bus, set during parse_args()
+ * */
+struct config{
+  struct target {
+	  int bus;
+  } target;
+} config = {
+	.target = {
+		.bus = -1
+	}
+};
+
+void help(void)
+{
+	printf(
+"\n"
+"fpga_dma_test \n"
+"Usage:\n"
+"\n"
+"               0          Hardware\n"
+"               1          Simulation only\n"
+"              -B, --bus   Set target bus number (OPTIONAL)\n"
+"\n"
+                   );
+}
+
+
+/*
+ * Parse command line arguments
+ */
+#define GETOPT_STRING "B"
+int parse_args(int argc, char *argv[])
```

Comment:
Use fpga_result as the return code. Return FPGA_OK on success, FPGA_EXCEPTION on failure